src/_tests/fixtures/43136/derived.json

```diff
@@ -3,7 +3,6 @@
   "now": "2020-03-19T19:14:42.000Z",
   "pr_number": 43136,
   "author": "larsrh",
-  "headCommitAbbrOid": "e686353",
```

Comment:
Redundant

Reply:
Maybe re-add it as part of the extended information, so less code needs to change?

Reply:
Done :heavy_check_mark: